commit 23e768dffa7712eaf96530d8cfca1c0f01dab8b9
Merge: f7ffd53 050ca9c
Author: Bo Herrmannsen <bo.herrmannsen@gmail.com>
Date:   Sun Dec 28 10:09:08 2014 +0100

    Merge pull request #1250 from chertykov/Development
    
    Move initialization of errormagic[] and echomagic[] to Marlin_main.cpp

diff --git a/Marlin/Marlin.h b/Marlin/Marlin.h
index e271ddc..444c43f 100644
--- a/Marlin/Marlin.h
+++ b/Marlin/Marlin.h
@@ -66,8 +66,9 @@
 #define SERIAL_PROTOCOLLNPGM(x) (serialprintPGM(PSTR(x)),MYSERIAL.write('\n'))
 
 
-const char errormagic[] PROGMEM ="Error:";
-const char echomagic[] PROGMEM ="echo:";
+extern const char errormagic[] PROGMEM;
+extern const char echomagic[] PROGMEM;
+
 #define SERIAL_ERROR_START (serialprintPGM(errormagic))
 #define SERIAL_ERROR(x) SERIAL_PROTOCOL(x)
 #define SERIAL_ERRORPGM(x) SERIAL_PROTOCOLPGM(x)
diff --git a/Marlin/Marlin_main.cpp b/Marlin/Marlin_main.cpp
index 6ea4ae3..809da79 100644
--- a/Marlin/Marlin_main.cpp
+++ b/Marlin/Marlin_main.cpp
@@ -329,6 +329,9 @@ bool cancel_heatup = false ;
   int meas_delay_cm = MEASUREMENT_DELAY_CM;  //distance delay setting
 #endif
 
+const char errormagic[] PROGMEM = "Error:";
+const char echomagic[] PROGMEM = "echo:";
+
 //===========================================================================
 //=============================Private Variables=============================
 //===========================================================================
